fe/fe-core/src/main/java/com/starrocks/common/ErrorCode.java

```diff
@@ -319,7 +317,15 @@ public enum ErrorCode {
     /**
      * 5300 - 5399: Lock and Transaction
      */
-    ERR_LOCK_ERROR(5300, new byte[] {'5', '5', 'P', '0', '3'}, "Attempt to acquire lock fail : %s");
+    ERR_LOCK_ERROR(5300, new byte[] {'5', '5', 'P', '0', '3'}, "Attempt to acquire lock fail : %s"),
+
+    /**
+     * 5400 - 5499: Internal error
+     */
+    ERR_CHANGE_TO_SSL_CONNECTION_FAILED(5300, new byte[] {'4', '2', '0', '0', '0'},
+            "Change to ssl connection failed"),
```

Comment:
1. Failed to change to SSL connection.
2. Is this a syntax error? (42000)
What is the scenario?

Reply:
1. done.
2. Not syntax error. This is the error happened on ssl handshake, maybe protocol not compatible.